lib/trivia_app.ex

```diff
@@ -5,18 +5,28 @@ defmodule TriviaApp do
   end
 
   @questions  [
-                %{ question: "Samite is a type of what: a) Fabric? b) Stone? c) Dog? d) Cake?", answer: "a" }, 
-                %{ question:  "Vermillion is a shade of which colour: a) Green? b) Blue? c) Red? d) Yellow?", answer: "c" }
+                %{ question: "Samite is a type of what: \na) Fabric \nb) Stone \nc) Dog \nd) Cake ", answer: "a" }, 
```

Comment:
This is pretty cool ☺️
The next stage in "extraction" is to figure out a file format, so you don't even need to re-build the application to add new questions 😄👍👏